deps/media-playback/media-playback/decode.c

```diff
@@ -230,8 +230,10 @@ bool mp_decode_init(mp_media_t *m, enum AVMediaType type, bool hw)
 		d->in_frame = d->sw_frame;
 	}
 
+#if LIBAVCODEC_VERSION_INT < AV_VERSION_INT(59, 8, 100)
```

Comment:
I'm pretty sure all of those could be simplified to `LIBAVCODEC_VERSION_MAJOR < 60`

Reply:
that's true, i see that in the commit Tytan has linked indeed